File: 2022.01.19/javascript/3009.js
const input = require('fs').readFileSync('/dev/stdin').toString().split('\n');
let x = [];
let y = [];

let a,b;
for (let i = 0 ; i<input.length-1 ; i++){
    [a,b] = input[i].split(' ').map(Number)
    x.push(a);
    y.push(b);
}

x = x[0] !== x[1] ? x[0] !== x[2] ? x[0] : x[1] : x[2];
y = y[0] !== y[1] ? y[0] !== y[2] ? y[0] : y[1] : y[2];
console.log(x,y);

File: Boj.단계별/9_기본수학2/python/3009.py
x = []
y = []
answer = [0,0]
for _ in range(3):
    a,b = map(int,input().split())
    x.append(a)
    y.append(b)
for i in range(len(x)):
    if x.count(x[i]) == 1 : answer[0] = x[i]
    if y.count(y[i]) == 1 : answer[1] = y[i]
print(answer[0],answer[1])